lms/djangoapps/certificates/tests/test_tasks.py

```diff
@@ -0,0 +1,21 @@
+from unittest import TestCase
+
+import ddt
+from mock import patch
+
+from lms.djangoapps.certificates.tasks import generate_certificate
+
+
+@ddt.ddt
+class GenerateUserCertificateTest(TestCase):
+    @patch('lms.djangoapps.certificates.tasks.generate_user_certificates')
+    def test_cert_task(self, generate_user_certs_mock):
+        generate_certificate(student='a', course_key='b', otherarg='c', otherotherarg='d')
```

Comment:
Nit: tasks are usually called via `apply_async()`, this test would be more real-world-comparable if you called that here. CELERY_ALWAYS_EAGER will convert it to a synchronous call so it's not *exactly* the same, but it's closer.